ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -6,6 +6,13 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "deleteCase",
```

Comment:
Should we change to 'deleteApplication' for consistency with the content?

Reply:
It makes sense since we have `submitApplication`. A `deleteApplication` would nicely fit it.